nebula-logger/core/main/configuration/customMetadata/LoggerParameter.StoreApexHeapSizeLimit.md-meta.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
I'm slightly second-guessing the dev name for the record - I think the word `Apex` is somewhat implied/not needed, so maybe the dev name could be shortened to just `StoreHeapSizeLimit`?